lighthouse-cli/bin.ts

```diff
@@ -281,6 +281,14 @@ function runLighthouse(url: string,
       return results;
     })
     .then((results: Results) => {
+      if (url !== results.url) {
```

Comment:
after #1677 I now realize you should do the comparison with `URL.equalWithExcludedFragments` so it doesn't warn on dropped fragments